storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/StorIOAnnotationsProcessor.java

```diff
@@ -98,10 +100,13 @@ private TypeElement validateAnnotatedClass(@NotNull final Element annotatedEleme
      * Checks that element annotated with {@link StorIOColumnMeta} satisfies all required conditions.
      *
      * @param annotatedElement an annotated field
+     * @return boolean flag which indicates if annotated element valid or not
      */
-    protected void validateAnnotatedFieldOrMethod(@NotNull final Element annotatedElement) {
+    protected boolean validateAnnotatedFieldOrMethod(@NotNull final Element annotatedElement) {
```

Comment:
I would prefer to rename this method or throw specific exception, because at the moment this code mixes two concepts `return value` and `exceptions` that define how to behave for the caller, either we need to convert everything to return values (ie enum instead of exceptions) or throw exception instead of return value